gammapy/morphology/overlap.py

```diff
@@ -175,7 +175,7 @@ def get_containment_mask(glon_pos, glat_pos, r_containment, shape):
     hdulist = fits.open('../counts.fits')
     w = WCS(hdulist[0].header)
     y, x = np.indices(shape)
-    glon, glat = w.wcs_pix2world(x, y, 1)
+    glon, glat = w.wcs_pix2world(x, y, 0)
```

Comment:
Ah, sorry, I missed this comment.

I thought `origin` was an explicit argument of `WCS.wcs_world2pix`, but now I see that the function signature is `wcs_pix2world(*args, **kwargs)` and if `origin` is passed as a keyword argument it's not handled by the function. I would say that's a bad API, but I don't think it makes sense to complain about this now that it's been like this for years.

So I would say leave as is then, just pass `0` as third parameter.

The alternatives would be to every time write
```python
origin = 0 # Astropy pixel coordinate convention
wcs.wcs_world2pix(x, y, origin)
```
or
```python
# Note: third parameter is `origin=0`, following the Astropy pixel convention
wcs.wcs_world2pix(x, y, 0)
```
which would be fine with me as well, but is a bit tedious.
@mapazarr – You decide.